content/en/blog/_posts/XXXX-XX-XX-In-Place-Pod-Resize-Beta.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: blog
+title: "In-Place Pod Resize Graduating to Beta"
+slug: in-place-pod-resize-beta
+canonicalUrl: https://www.kubernetes.dev/blog/XXXX/XX/XX/in-place-pod-resize-beta
+date: XXXX-XX-XX
+author: "Tim Allclair"
+---
+
+## In-Place Pod Resize graduates to Beta in Kubernetes v1.33!
+
+We are excited to announce that the **In-Place Pod Resizing** feature (also known as In-Place Pod Vertical Scaling), first introduced as alpha in Kubernetes v1.27, is graduating to **Beta** and will be enabled by default in the Kubernetes v1.33 release! This marks a significant milestone in making resource management for Kubernetes workloads more flexible and less disruptive.
+
+### What is In-place Pod Resize?
+
+Traditionally, changing the CPU or memory resources allocated to a container required restarting the Pod. While acceptable for many stateless applications, this could be disruptive for stateful services, batch jobs, or any workloads sensitive to restarts.
+
+In-place pod resize allows you to change the CPU and memory requests and limits assigned to containers within a *running* Pod, often without requiring a container restart.
+
+Here's the core idea:
+* The `spec.containers[*].resources` field in a Pod specification now represents the *desired* resources and is mutable for CPU and memory.
```

Comment:
Do we want to mention `AllocatedResources`? It might be useful for development of end-user o11y tools

Reply:
I decided not to because I think most users should just look at `resources`. The documentation page (see https://github.com/kubernetes/website/pull/50290) discusses `allocatedResources` in more detail. WDYT?

Reply:
SG.